Error buildling develop_2.0

Executed "mvn clean && ./bin/php-petstore.sh" but got the following error

```
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.269 sec

Results :

Tests in error: 
  should check handling of recursive models(ExampleGeneratorTest): "<Pair>[

Tests run: 51, Failures: 0, Errors: 1, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] swagger-codegen-project ........................... SUCCESS [2.570s]
[INFO] swagger-codegen (core library) .................... FAILURE [1:21.930s]
[INFO] swagger-codegen (executable) ...................... SKIPPED
[INFO] swagger-generator ................................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1:26.460s
[INFO] Finished at: Thu May 21 04:44:54 HKT 2015
[INFO] Final Memory: 23M/69M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.10:test (default-test) on project swagger-codegen: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/apps/wing328/swagger-codegen/modules/swagger-codegen/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :swagger-codegen
Error: Unable to access jarfile ./modules/swagger-codegen-cli/target/swagger-codegen-cli.jar
```

--- a/modules/swagger-codegen/src/test/scala/ExampleGeneratorTest.scala
+++ b/modules/swagger-codegen/src/test/scala/ExampleGeneratorTest.scala
@@ -42,17 +42,17 @@ class ExampleGeneratorTest extends FlatSpec with Matchers {
       item.get("contentType") match {
         case `xml` => {
           types += xml
-          example should be ("<Pair>\\n" +
-                             "  <Node>\\n" +
-                             "    <name>string</name>\\n" +
-                             "    <wrappedChildren>\\n" +
-                             "    </wrappedChildren>\\n" +
-                             "  </Node>\\n" +
-                             "  <Node>\\n" +
-                             "    <name>string</name>\\n" +
-                             "    <wrappedChildren>\\n" +
-                             "    </wrappedChildren>\\n" +
-                             "  </Node>\\n" +
+          example should be ("<Pair>\n" +
+                             "  <Node>\n" +
+                             "    <name>string</name>\n" +
+                             "    <wrappedChildren>\n" +
+                             "    </wrappedChildren>\n" +
+                             "  </Node>\n" +
+                             "  <Node>\n" +
+                             "    <name>string</name>\n" +
+                             "    <wrappedChildren>\n" +
+                             "    </wrappedChildren>\n" +
+                             "  </Node>\n" +
                              "</Pair>")
         }
         case `json` => {

